packages/fxa-settings/src/components/ButtonBack/index.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { ReactComponent as BackArrow } from './back-arrow.svg';
+import { useFtlMsgResolver } from '../../models';
+
+interface ButtonBackProps {
+  onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  dataTestId?: string;
+  localizedLabel?: string;
+}
+
+// This component assumes a parent container with class 'relative flex'
+// and an adjacent text element.
+const ButtonBack = ({
+  onClick = () => window.history.back(),
+  dataTestId,
+  localizedLabel,
+}: ButtonBackProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+  const localizedTitle =
+    localizedLabel || ftlMsgResolver.getMsg('button-back-title', 'Back');
+  const localizedAriaLabel =
```

Comment:
Should there also be a distinct param for localizedLabelAria? This would be symmetric with how the l10n stings are setup.

Reply:
Yeah, I'll go ahead and do that. I don't foresee a case where these are ever different but it's probably better to go ahead and do it this way, thanks!